index.html

```diff
@@ -23,25 +23,244 @@ <h1><br>I’m George<br>Glad to see you!</h1>
+        <nav class="nav-icon">
+          <ul>
+            <li>
+              <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/linkedin.svg" alt="Linkedin" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.svg" alt="GitHub" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/angellist.svg" alt="Angellist"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.svg" alt="Medium" /></a>
+            </li>
+          </ul>
+        </nav>
+        <div class="get-my-resume">
+          <p class="button"><a href="#">Get My Resume</a></p>
+        </div>
+        <div class="outside">
+          <a href="#" class="skills-list">
+            <img src="./images/skills-list.svg" alt="List of Skills" />
+                </a>
+              </div>
+             </section>
+             <section>
+              <div class="contact-me">
+                <div class="contact-wrapper">
+                  <h2>Contact Me</h2>
+                  <p>
+                    If you have an application you are interested in developing a<br>
+                    feature that you need built or a project that needs coding. I’d
+                    love to help with it
+                  </p>
+                  <form action="https://formspree.io/f/meqngryy" method="post">
+                    <ul>
+                      <li>
+                        <input
+                          type="text"
+                          class="me"
+                          id="me"
+                          name="contactme"
+                          maxlength="30"
+                          placeholder="George D. Kruah"
+                          required
+                        />
+                      </li>
+                      <li>
+                        <input
+                          type="email"
+                          class="email"
+                          id="email"
+                          name="email"
+                          placeholder="dfizzkruah@gmail.com"
+                          required
+                        />
+                      </li>
+                      <li>
+                        <textarea
+                          name="message"
+                          class="message"
+                          maxlength="500"
+                          id="message"
+                          placeholder=""
+                          required
+                        ></textarea>
+                      </li>
+                      <li>
+                        <button type="submit" name="get-in-touch">
+                          <strong>Get in Touch</strong>
+                        </button>
+                      </li>
+                    </ul>
+                  </form>
```

Comment:
- Please add placeholder text to the `textarea` field to match the Figma design.
- Kindly adjust the design to be more like the Figma design. You can use pseudo classes for the placeholder to change its font and styling ( [Check this MDN page to know how to use this feature ](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)).
- Kindly adjust the background image of this section to match the Figma design. The background doesn't appear in the design, you can fix this by using `./images/header-mobile.svg` instead of `/images/header-mobile.svg` in the image URL path.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155294787-7c89df46-fad0-487d-b3c0-c6ac1c5f6220.png) | ![image](https://user-images.githubusercontent.com/24703171/155294919-3623164e-4a76-4cc7-81ec-efdbeb13faf3.png) |